deploy/chart/values.yaml

```diff
@@ -1,4 +1,3 @@
-installType: upstream
```

Comment:
Why is this removed?

Reply:
This was duplicated - the same key/value is down below